### PR TITLE
Implements pool for postgreSQL and data-generator script

### PR DESCRIPTION
**Pull request by DavidFuent1**
Medium Sized PR focused on getting PostgreSQL up and running with data-generator script (independent of actual seeding) 

Look out for:

1) .gitignore -- added csv file
3) server/app.js -- comments out existing restful API/references new db connection for PostgreSQL and adds an app.get() 
6) server/db/data-generator.js -- implements async data generator function  
7) server/postgreSQL/ -- added connection.js (connection for postgreSQL)

### CODE REVIEW

File: server/postgreSQL/connection.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const Pool = require('pg').Pool;
+
+const pool = new Pool({
+  user: 'dfuent',
+  host: 'localhost',
+  database: 'movies',
+  password: '',
+  port: 5432
+});
```
Comment (mmcphillips) [inline server/postgreSQL/connection.js]: Implement a dotenv to avoid hard coding credentials

File: server/db/data-generator.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    encoding: 'utf8'
+  }
+);
+
+const generator = () => {
+  (async () => {
+    movieCSV.write('movie_id,cast_id, actor, character, image_url, role\n');
+    for (var i = 0; i < generated; i++) {
+      if (i % 24 === 0) {
+        movieIdCounter++;
```
Comment (mmcphillips) [inline server/db/data-generator.js]: Avoid ++ operators to adhere to airbnb styleguide

File: server/db/data-generator.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+const movieCSV = fs.createWriteStream(
+  '/Users/dfuent/Desktop/SDC MODULE/castphotos/movies.csv',
+  {
+    encoding: 'utf8'
+  }
+);
+
+const generator = () => {
+  (async () => {
+    movieCSV.write('movie_id,cast_id, actor, character, image_url, role\n');
```
Comment (RachNovo) [inline server/db/data-generator.js]: Forgot a space here. @DavidFuent1

File: server/app.js [hunk 4]
```diff
@@ -12,39 +13,52 @@ app.use(bodyParser.urlencoded({ extended: true }));
...
+app.get('/api/movie', (req, res, next) => {
+  const { id } = req.query;
+  pool.query(`SELECT * FROM movies WHERE movie_id = ${id};`, (err, results) => {
+    if (err) {
+      res.sendStatus(404);
+    } else {
+      res.status(200).send(results);
+    }
+  });
 });
```
Comment (RachNovo) [inline server/app.js:62]: Why is this query in your server file? Wouldn't it be better in a database file? @DavidFuent1

Comment (DavidFuent1) [reply server/app.js:62]: @RachNovo I plan on moving it over for modularity, was just trying to get a path in for testing.

File: server/db/data-generator.js (hunk 2 above)
Comment (DavidFuent1) [reply server/db/data-generator.js]: @mmcphillips refactored

File: server/postgreSQL/connection.js (hunk 1 above)
Comment (DavidFuent1) [reply server/postgreSQL/connection.js]: @mmcphillips dotenv.config() being used now

File: server/db/data-generator.js (hunk 3 above)
Comment (DavidFuent1) [reply server/db/data-generator.js]: @RachNovo refactored :)

**Pull request merged**